crates/biome_service/tests/valid/deprecated_options_inside_javascript.json

```diff
@@ -7,7 +7,7 @@
 		"formatter": {
 			"enabled": true,
 			"indentSize": 2,
-			"trailingComma": "all"
+			"trailingCommas": "all"
```

Comment:
We shouldn't change it because this is for testing purpose. We need to ensure the deprecated `trailingComma` is valid.